Prox/Prox/Data/PlacesController.swift

```diff
@@ -40,6 +40,9 @@ class PlacesProvider {
         return FIRRemoteConfig.remoteConfig()[key].numberValue!.doubleValue
     }()
 
+    private var nearbyPlaces = [CLLocation: [Place]]()
```

Comment:
We are only caching for the duration of the call. Once we have received a response from both the nearby place search and the places with events search then the cache is cleared. It is there solely to enable us to merge the two sets of places together once we have them.